captum/optim/_param/image/transforms.py

```diff
@@ -36,6 +47,13 @@ class IgnoreAlpha(nn.Module):
     r"""Ignores a 4th channel"""
 
     def forward(self, x: torch.Tensor) -> torch.Tensor:
+        """
+        Ignore the alpha channel.
+        Arguments:
+            x (torch.Tensor): RGBA image tensor.
+        Returns:
+            rgb (torch.Tensor): RGB image tensor without the alpha channel.
+        """
         assert x.dim() == 4
```

Comment:
Since `klt_transform` and `i1i2i3_transform` are public, it would be good to document them.
`__ init__` for `ToRGB ` needs docs as well.

Reply:
Okay, I've add some documentation and moved the init args from the main description to the init function.